ui/app/src/state/reducers/dialogs/publishingStatus.ts

```diff
@@ -39,7 +39,11 @@ const initialState: PublishingStatusDialogStateProps = {
   isFetching: false,
   onClose: closePublishingStatusDialog(),
   onRefresh: fetchPublishingStatus(),
-  onUnlock: null
+  onUnlock: null,
+  numberOfItems: null,
+  totalItems: null,
+  environment: null,
```

Comment:
Please rename `environment` to `publishingTarget`. Abstract away in the service as this will change in the api.